sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -348,4 +377,61 @@ void populateDefaults(AbstractTelemetryBuilder builder, Resource resource) {
             ContextTagKeys.AI_INTERNAL_SDK_VERSION.toString(), VersionGenerator.getSdkVersion());
         ResourceParser.updateRoleNameAndInstance(builder, resource, configuration);
     }
+
+    /**
+     * Creates an {@link AutoConfiguredOpenTelemetrySdkBuilder} based on the options set in the builder.
+     *
+     * @return An instance of {@link AutoConfiguredOpenTelemetrySdkBuilder}.
+     */
+    public AutoConfiguredOpenTelemetrySdkBuilder getOpenTelemetrySdkBuilder() {
+        return AutoConfiguredOpenTelemetrySdk.builder()
+            .addPropertiesSupplier(() -> {
+                Map<String, String> props = new HashMap<>();
+                props.put("otel.traces.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+                props.put("otel.metrics.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+                props.put("otel.logs.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+                props.put(AzureMonitorExporterProviderKeys.INTERNAL_USING_BUILDER, "true");
+                return props;
+            })
+            .addSpanExporterCustomizer(
+                (spanExporter, configProperties) -> {
+                    if (spanExporter instanceof AzureMonitorSpanExporterProvider.MarkerSpanExporter) {
+                        spanExporter = buildTraceExporter(configProperties);
+                    }
+                    return spanExporter;
+                })
+            .addMetricExporterCustomizer(
+                (metricExporter, configProperties) -> {
+                    if (metricExporter instanceof AzureMonitorMetricExporterProvider.MarkerMetricExporter) {
+                        metricExporter = buildMetricExporter(configProperties);
+                    }
+                    return metricExporter;
+                })
+            .addLogRecordExporterCustomizer(
+                (logRecordExporter, configProperties) -> {
+                    if (logRecordExporter instanceof AzureMonitorLogRecordExporterProvider.MarkerLogRecordExporter) {
+                        logRecordExporter = buildLogRecordExporter(configProperties);
+                    }
+                    return logRecordExporter;
+                })
+            .addMeterProviderCustomizer((sdkMeterProviderBuilder, configProperties) -> {
+                sdkMeterProviderBuilder.registerView(
+                    InstrumentSelector.builder()
+                        .setMeterName("io.opentelemetry.sdk.trace")
```

Comment:
should we make meter name configurable? in the old way, customer sets it. it would be nice to make it configurable by customers?